client/src/client-only-routes/show-settings.tsx

```diff
@@ -154,7 +153,7 @@ export function ShowSettings(props: ShowSettingsProps): JSX.Element {
     navigate(`${apiLocation}/signin`);
     return <Loader fullScreen={true} />;
   }
-
+  const sound = store.get('fcc-sound') as boolean;
```

Comment:
```suggestion
  const sound = store.get('fcc-sound') ?? false;
```

Rather than asserting, we should probably coalesce.

Reply:
done